examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -210,10 +213,11 @@ def forward(self, emission: torch.Tensor) -> str:
         indices = torch.argmax(emission, dim=-1)  # [num_seq,]
         indices = torch.unique_consecutive(indices, dim=-1)
         indices = [i for i in indices if i != self.blank]
-        return "".join([self.labels[i] for i in indices])
+        joined = "".join([self.labels[i] for i in indices])
+        return joined.replace("|", " ").lower().strip().split()
```

Comment:
Can you update the return type of the signature and docstring?

Reply:
I think `lower` is redundant as the `tokens` passed to the constructor is already lowered.